test/statetest/statetest_runner.cpp

```diff
@@ -0,0 +1,47 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2022 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "../state/mpt_hash.hpp"
```

Comment:
Better `<test/state/mpt_hash.hpp>` ?

Reply:
Doesn't work. I probably would need to specify additional include paths what does not sound worth it.